### ISSUE TITLE
Further enhancements to Preview to ensure that users never have to manually clear cache to see the latest changes

### ISSUE THREAD

**Issue by alcurrie** [ampproject/amp-wp#1731]
As an AMP Story author, I want to always see my newest changes in the Preview with having to clear cache

_Background:_ this issue to further iterate on the issue discuss in #1570  that users building AMP stories in the Pantheon demo environment frequently had to clear cache to see an updated preview. However, since this issue is not repeatable in a local environment but only on the Pantheon demo site, it is not clear whether or how to resolve. This issue is to research and if needed make changes to resolve. 

Related to: #1570

**Comment by westonruter** [comment]
I looked into this again and I found something interesting. On our story-test environment, even with AMP plugin disabled entirely, the issue is present when looking at previews for Posts. If I have DevTools open (with Disk Cache not disabled) in the preview window and I make another change in the editor and click preview again, I see something unexpected:

![image](https://user-images.githubusercontent.com/134745/58933806-4ac90480-871d-11e9-9d94-18fedadd56aa.png)

Notice `(from disk cache)`. That should not be! It should always be going to the network.
